### ISSUE TITLE
Iterations in RANSAC

### ISSUE THREAD

**Issue by alexs7** [willGuimont/PROSAC#1]
Hello, great work! I was just wondering if you change the times of iterations in ransac ?

Now you have this `max_times = int(np.ceil(1 / (prob_inlier ** m)))` but that doesn't change in the RANSAC loops

**Comment by willGuimont** [comment]
`max_times` is an upper bound on the number of times you have to draw samples. Knowing that each point has `prob_inlier` of being an inlier, so the probability of having all inliers is `prob_inlier ** m`. Then we find the expected number of draws to get a set of point entirely composed of inliers.

Is it clearer?

**Comment by alexs7** [comment]
@willGuimont  Yes it helps. I have used a different approach for this. I have followed this : https://www.youtube.com/watch?v=9D5rrtCC_E0 (at 3:28) - So the max iterations are set to a number in the beginning and then I change then according to that formula in the video.

My code is here: https://github.com/alexs7/Mobile-Pose-Estimation-Pipeline-Prototype/blob/demo_version/ransac.py (method run_ransac())
